src/js/components/LayerPanel/FiresControls.js

```diff
@@ -101,10 +107,11 @@ export default class FiresControls extends React.Component {
           <div
             style={{border: `1px solid ${customColorTheme && customColorTheme !== '' ? customColorTheme : defaultColorTheme}`}}
             className="fa-button sml white pointer"
-            onClick={() => this.setState({
-              customRange: !customRange,
-              activeFireOptionLabel: 'Active Fires'
-            })}
+            onClick={() => {
+                layerActions.updateCustomRange(!this.state.customRange);
+                layerActions.updateActiveFireOptionLabel('Active Fires');
```

Comment:
@KaylaKremer We only want the `label` to change if the **layer** on the map is `visible` or not. Can we base that on the `visibility` of the **1YR** layer?

Reply:
I fixed it but used a different approach! Also added some disabling and styling so user won't think they can click on the dropdown while they're using custom range (calendar).